### PR TITLE
First draft of changelog for 2.3.0; focuses on significant changes

### PR DESCRIPTION
**Pull request by mpacer**
This could probably use some word-smithing. But here's a first pass at the changelog.

I focused on significant changes rather than cataloguing the different PRs. If we want to I can link the PRs from their relevant areas of focus.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,22 +2,65 @@
 
 ## 2.3.0 [Unreleased](https://github.com/nteract/bookstore/compare/2.2.1...HEAD)
 
-[2.3.0 on Github](https://github.com/nteract/bookstore/releases/tag/2.3.0)
+## [2.3.0](https://github.com/nteract/bookstore/releases/tag/2.3.0) 2019-07-02
+
+The following 6 authors contributed X commits.
```
Comment (willingc) [inline CHANGELOG.md]:
```suggestion
Thank you to the following contributors:
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,22 +2,65 @@
...
-Validation information is now exposed as a dict at the `/api/bookstore` endpoint.
+#### New Publishing endpoint
+
+Previously our publishing endpoint was `/api/bookstore/published`, it is now `/api/bookstore/publish`.
+
+#### Cloning
+
+As of 2.3.0 cloning from S3 is now enabled by default.
+
+Cloning allows access to multiple buckets, but you will need to have your configuration set up for any such bucket.
```
Comment (willingc) [inline CHANGELOG.md]:
```suggestion
Cloning allows access to multiple S3 buckets. To use them, you will need to set up your configuration for any such bucket.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,22 +2,65 @@
...
+
+#### Massive Testing improvements
+
+We have built out a framework for unit-testing Tornado handlers. In addition, we have added a collection of unit tests that bring us to a coverage level in non-experimental code of well over 80%.
```
Comment (willingc) [inline CHANGELOG.md]: Woo!!!

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,22 +2,65 @@
...
-This allows us to distinguish whether different features have been enabled on bookstore.
+You can identify which features have been enabled and which version of bookstore is available by using the `/api/bookstore` endpoint.
+
+#### REST API Documentation
+
+All APIs are now documented at our [REST API docs](https://bookstore.readthedocs.io/en/latest/openapi.html) using the OpenAPI spec.
+
+#### Clients (Experimental Feature)
+
+In order to enable access to bookstore publishing and cloning we have created a Notebook and Bookstore clients. *This is not well-tested* functionality at the moment, so we discourage its use in production.
```
Comment (willingc) [inline CHANGELOG.md]:
```suggestion
To enable access to bookstore publishing and cloning from within a notebook, we have created a Notebook and Bookstore clients. *This is still experimental* functionality at the moment and needs additional testing, so we discourage its use in production.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,22 +2,65 @@
...
+It relies on an assumption that a single kernel is attached to a single notebook, and will break if you use multiple notebooks attached to the same kernel.
```
Comment (willingc) [inline CHANGELOG.md]:
```suggestion
The design relies on an assumption that a single kernel is attached to a single notebook, and will break if you use multiple notebooks attached to the same kernel.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,22 +2,65 @@
...
+
+However, for those who wish to experiment, it offers some fun ways of exploring bookstore.
+
+For example, if you were to run a notebook from within the top-level [bookstore/ci](https://github.com/nteract/bookstore/tree/master/ci) directory while running the integration test server with `yarn test:server` (see more about [local integration testing](https://bookstore.readthedocs.io/en/latest/project/local_ci.html)).
```
Comment (willingc) [inline CHANGELOG.md]:
```suggestion
Example: if you run a notebook from within the top-level [`bookstore/ci`](https://github.com/nteract/bookstore/tree/master/ci) directory while running the integration test server with `yarn test:server` (see more about [local integration testing](https://bookstore.readthedocs.io/en/latest/project/local_ci.html)),
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,22 +2,65 @@
...
+You should be able to publish from inside a notebook using the following code snippet.
```
Comment (willingc) [inline CHANGELOG.md]:
```suggestion
you should be able to publish from inside a notebook using the following code snippet:```

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,22 +2,65 @@
...
+#### `/api/bookstore/`: Features and Versions
 
-This allows us to distinguish whether different features have been enabled on bookstore.
+You can identify which features have been enabled and which version of bookstore is available by using the `/api/bookstore` endpoint.
+
+#### REST API Documentation
+
+All APIs are now documented at our [REST API docs](https://bookstore.readthedocs.io/en/latest/openapi.html) using the OpenAPI spec.
+
+#### Clients (Experimental Feature)
```
Comment (willingc) [inline CHANGELOG.md]:
```suggestion
### Experimental
#### Clients (subject to change in future releases)
```

**Comment by mpacer** [comment]
Merging!

**Pull request merged**